Header header 
float32[] Var_map
float32[] LSD_map
sensor_msgs/Image coloredImage
float32[] cameraMatrix
float32[] RotationMatrix
float32[] translationVector

